Mathlib/Algebra/GroupWithZero/Power.lean

```diff
@@ -15,6 +15,25 @@ In this file we define integer power functions for groups with an adjoined zero
 This generalises the integer power function on a division ring.
 -/
 
+section MonoidWithZero
+
+variable {M₀ : Type _} [MonoidWithZero M₀] {x y : M₀} {k : ℕ}
+
+/-- If `x` is not a (left) zero divisor then neither is `x^k` for any `k`. -/
```

Comment:
Isn't this true for multiplication on the left by *any* element, not just powers of `x`?